docs/mongocli/command/mongocli-iam-organizations-apiKeys-accessLists-create.txt

```diff
@@ -44,6 +44,10 @@ Options
      - strings
      - false
      - Access list entry in CIDR notation to be added for your API key. To add more than one entry, you can specify each entry with a separate cidr flag or specify all the entries as a comma-separated list using one cidr flag. You can't set both cidr and ip in the same command.
+   * - --currentIp
+     - 
```

Comment:
empty line ?

Reply:
yes, this renders as a table, this is a column, it'll have no value, if you want to know more these docs are auto-generated from the code using https://github.com/mongodb-labs/cobra2snooty